pkg/svc/extension.go

```diff
@@ -406,82 +413,64 @@ func (s *extensionRegistrySvc) DeleteService(ctx context.Context, req *extension
 // extension registry.
 func (s *extensionRegistrySvc) AddEndpoint(ctx context.Context, req *extension.ExtensionEndpoint) (res *extension.ExtensionEndpoint, err error) {
 	s.logger.Print("extension.addEndpoint")
-	endpoint, err := s.registry.AddEndpoint(ctx, extensionEndpointToDomain(req))
+	endpoint, err := s.registry.AddEndpoint(ctx, *req.ExtensionID, *req.ServiceID, extensionEndpointToDomain(req))
```

Comment:
what if the extension ID or the service ID are nil ?

Reply:
Fixed by using util.DerefString